be/tool/beck.c

```diff
@@ -1167,21 +1256,34 @@ static int emap_proc(struct scanner *s, struct btype *btype,
 		ret = emap_kv_get(s, &node->bt_kv_arr[i],
 				  &ea->emap_key, &ea->emap_val);
 		if (ret != 0) {
+			if (s->s_print_invalid_oids) {
+				ek = ea->emap_key.b_addr;
+				inv_emap_off = node->bt_kv_arr[i].btree_key -
+					       s->s_seg->bs_addr;
+				emap_to_gob_convert(&ek->ek_prefix, &gob_id);
+				M0_LOG(M0_ERROR, "Found incorrect EMAP entry at"
+				       " segment offset %"PRIx64" for GOB "
+				       FID_F, inv_emap_off, FID_P(&gob_id));
```

Comment:
At this moment, the `ea->emap_key.b_addr` may contain invalid data.
Or maybe it is NULL.
If it just contains some garbage, the emap_to_gob_convert() will work properly?